server/controllers/meeting_controller.coffee

```diff
@@ -1,6 +1,8 @@
 Meeting     = require "../models/meeting"
 User        = require "../models/user"
 googleAuth  = require "../helpers/auth/google"
+googleapi   = require 'googleapis'
```

Comment:
Appears to be unused (also should only be required in "helpers/auth/google" now)